pages/01.HDM Documentation/06.vcd/05.Deployment Planning/default.md

```diff
@@ -104,3 +104,46 @@ _Table 1: Resource Requirements_
 1. The names _ESXMgr_ and _CloudCache_ are abbreviated names for the VMs containing HDM components. These are created as part of the HDM deployment.
 2. The N in _Cluster (N)_ refers to the number of nodes in the cloud cluster.
 
+
+### Prepare Cloud for Deployment
+
+To deploy the HDM solution on  _IBM Cloud for VMware Solutions Shared_ you would
+need to
+
+* Get the _Organization Administrator_ access credentials to the VCD tenant portal.
+* Identify the organization ID and save for later use.
+* Create a new user with admin privilege.
+
+**Identify the oprganization ID**
+
+* Loging to the tenant portal
+* The Organization ID can be identified from the location in  _Figure 1_
+
+_Figure 1: Screenshot show how to identify Organization ID_
+![Organization Id](images/Organization-id.png?classes=content-img "Screenshot showing how to find organization id") 
+
+
+**Create new user with Organization Administrator priveleges**
+
+We recommend the creation of a new organization administrator for HDM integration.
+We refer to this administrator in the rest of the documentation.
+
+1. Create the new admin user using procedure [here to create new user.](https://docs.vmware.com/en/VMware-Cloud-Director/10.1/VMware-Cloud-Director-Tenant-Portal-Guide/GUID-1CACBB2E-FE35-4662-A08D-D2BCB174A43C.html) 
+1. Make sure you give it the **Organization Administrator** role see _Figure 2._
```

Comment:
Point numbering (1) repeated.

Reply:
Ayyaj in MD a numbered list is written as

1. asdfsaf
1. asdfasf

MD renders them numerically, see the rendering in 

http://ec2-34-222-41-154.us-west-2.compute.amazonaws.com/hdm%20documentation/vcd/deployment%20planning

<img width="1344" alt="Screen Shot 2020-11-06 at 3 26 44 PM" src="https://user-images.githubusercontent.com/391442/98352879-997e7a80-2044-11eb-9bcd-57116dc40f98.png">